### PR TITLE
Fixes crash + camera orientation

### PR DESCRIPTION
**Pull request by ata-peppered**

### CODE REVIEW

File: Censo/PhotoCaptureController.swift [hunk 1]
```diff
@@ -92,13 +88,30 @@ class PhotoCaptureController: NSObject, ObservableObject {
...
+
                 let settings = AVCapturePhotoSettings()
                 photoOutput.capturePhoto(with: settings, delegate: self)
             }
         default:
             break
         }
     }
+
+    private func orienation() -> AVCaptureVideoOrientation {
```
Comment (bflood-strike) [inline Censo/PhotoCaptureController.swift:103]: orientation is misspelled here in the func name

Comment (bholzman) [inline Censo/PhotoCaptureController.swift:103]: nit: typo

**Review by bflood-strike — APPROVED** [review]
looks good to me - just one minor comment re spelling.

File: Censo/Views/DeviceRegistration/CameraPreview.swift [hunk 2]
```diff
@@ -11,13 +11,36 @@ import AVFoundation
...
+
         override class var layerClass: AnyClass {
             AVCaptureVideoPreviewLayer.self
         }
 
         var videoPreviewLayer: AVCaptureVideoPreviewLayer {
             layer as! AVCaptureVideoPreviewLayer
         }
+
+        @objc func orienationChanged() {
```
Comment (bholzman) [inline Censo/Views/DeviceRegistration/CameraPreview.swift:32]: nit: typo

**Review by bholzman — APPROVED** [review]
Looks good!

**Pull request merged**